src/toil/test/provisioners/aws/awsProvisionerTest.py

```diff
@@ -372,6 +372,13 @@ def testSpotAutoScale(self):
         self.numWorkers = ['2']
         self._test(preemptableJobs=True)
 
+    @integrative
+    @needs_aws_ec2
+    def testSpotAutoScaleBalancingTypes(self):
+        self.instanceTypes = ["m5.large;m5a.large:%f,t2.large:0.01" % self.spotBid]
```

Comment:
Is this supposed to be semicolon separated?

Reply:
Nope!